src/apply-preserving-inline-style.js

```diff
@@ -44,13 +61,19 @@
     WEB_ANIMATIONS_TESTING && console.assert(!(element.style instanceof AnimatedCSSStyleDeclaration),
         'Element must not already have an animated style attached.');
 
+    this._element = element;
     // Stores the inline style of the element on its behalf while the
     // polyfill uses the element's inline style to simulate web animations.
     // This is needed to fake regular inline style CSSOM access on the element.
     this._surrogateStyle = document.createElementNS('http://www.w3.org/1999/xhtml', 'div').style;
     this._style = element.style;
     this._length = 0;
     this._isAnimatedProperty = {};
+    this._updateSvgTransformAttr =
+        element.namespaceURI &&
+        element.namespaceURI.indexOf('/svg') != -1 &&
```

Comment:
Make element a parameter to the helper function instead of having the logic split up like this.